dluhc-component-library/src/components/maps/DatasetLayer.tsx

```diff
@@ -20,13 +23,29 @@ const DatasetLayer = ({ area, dataset, zIndex = 1 }: DatasetlayerProps) => {
     return null;
   }
 
+  const stroke: StrokeOptions = useMemo(() => {
+    const color = dataset["paint-options"]
+      ? dataset["paint-options"].colour
+      : "#003078";
+    return { color, width: 2 };
+  }, [dataset]);
+
+  const fill: FillOptions = useMemo(() => {
+    const paintOptions = dataset["paint-options"];
+    if (!paintOptions) {
+      return { color: "rgba(0, 48, 120, 0.2)" };
```

Comment:
`Non-blocker` pull this out into a constant as the default value, same with the 0.2 opacity below and the 2 width